flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/ExecutionGraphRestartTest.java

```diff
@@ -549,8 +538,8 @@ public void testGlobalFailAndRestarts() throws Exception {
 		eg.waitUntilTerminal();
 		assertEquals(JobStatus.FINISHED, eg.getState());
 
-		if (eg.getNumberOfFullRestarts() > 2) {
-			fail("Too many restarts: " + eg.getNumberOfFullRestarts());
+		if (eg.getNumberOfRestarts() > 2) {
+			fail("Too many restarts: " + eg.getNumberOfRestarts());
```

Comment:
Let's also replace this with an assertion.

Reply:
done